tasks/pages/properties.tsx

```diff
@@ -45,13 +49,14 @@ export const PropertyContext = createContext<PropertiesContextType>({
  */
 const PropertiesPage: NextPage = ({ overwriteTranslation }: PropsForTranslation<OrganizationsPageTranslation>) => {
   const translation = useTranslation(defaultOrganizationsPageTranslation, overwriteTranslation)
-  const propertyId = useRouteParameters<never, 'id'>().id
+  const { id: propertyId, subject: subjectType } = useRouteParameters<never, 'id' | 'subject'>()
```

Comment:
I cannot set these via the params
https://pr-932-helpwave-web-tasks.fly.dev/properties?subject=patient

Reply:
You have the wrong branch id, it should be [https://pr-931-helpwave-web-tasks.fly.dev/properties?subject=patient](https://pr-931-helpwave-web-tasks.fly.dev/properties?subject=patient)